app/src/main/java/com/babylon/wallet/android/presentation/wallet/WalletViewModel.kt

```diff
@@ -62,6 +65,8 @@ import rdx.works.profile.domain.display.ChangeBalanceVisibilityUseCase
 import timber.log.Timber
 import javax.inject.Inject
 
+private const val DELAY_BETWEEN_POP_UP_SCREENS_MS = 1000L
```

Comment:
however why such a long delay?

Reply:
it's purely a UI/UX thing, for the transitions to feel smoother

Reply:
anyone should feel free to adjust this value though